### ISSUE TITLE
[MyMeta] "Fixing" the Patrons leaderboard

### ISSUE THREAD

**Issue by peth-yursick** [MetaFam/TheGame#1201]
I put in quotation marks because its not really broken, just not reflecting Polygon pSeeds, which is where majority of holders currently are...

@polimyl made [this dynamic patrons ranked league sheet](https://docs.google.com/spreadsheets/d/1AYYidq8CRV7EbuimTCFMvz75XSsaFPq05gaqfSeozTU/edit?usp=sharing), which got me thinking we should do the same for [the patrons leaderboard](https://my.metagame.wtf/patrons).

What the sheet does, in layman terms:
- Takes in [Polygon pSeed holdings data](https://polygonscan.com/token/0x8a8fcd351ed553fc75aecbc566a32f94471f302e) & calculates it into dollar terms.
- Takes in [mainnet pSeed holdings data](https://etherscan.io/token/0x34a01c0a95b0592cc818cd846c3cf285d6c85a31) & calculates it into dollar terms.
- Takes in [yeeted eth data](https://yeet.daohaus.club/dao/0x1/0x8b9280f30f386c4b8fc363d5984845d946a936c3/1) & calculates it into dollar terms.
- Merges all of this $ amounts & holder address data.
- Figures out what are $ requirements for each ranked league.

Taking it a step further for the whole leaderboards (as here, the purpose was only to figure out rank requirements), it would also need to sort all of these holder addresses based on how much is each holding in dollar terms.

Idk what sort of wizardry was required to make the sheet & I know even less about what's required to encode this into MyMeta but I hope I at least explained it well enough?

## Why is this needed?
Firstly, because right now its only displaying mainnet holdings.
Accounting for Polygon pSeeds would help, but it still wouldn't to justice to the people who yeeted as their rank would only be updated once their eth is put inside the pool, which is months away + requirements likely to change by then.

Overall, the patrons leaderboard is needed to gamify patronage, thus adding an incentive for patrons to buy Seeds.

**Comment by peth-yursick** [comment]
Hey @arsenijesavic, any update on this?

**Comment by arsenijesavic** [comment]
@peth-yursick I will be working on this issue during this week, last week i was busy with hackathon and other projects!

**Comment by peth-yursick** [comment]
> @peth-yursick I will be working on this issue during this week, last week i was busy with hackathon and other projects!

Hey, were you able to make any progress? Need some help or just gave up when the token shit itself? 😂

**Comment by arsenijesavic** [comment]
@peth-yursick pushing this week :)

**Comment by peth-yursick** [comment]
You said you're down to do it, so I unassigned Arsenije (who had already given up 2 weeks earlier) & assigned you @alalonde.
If you changed your mind meanwhile, please unassign & I'll try to find somebody else :)
